cmd/scollector/collectors/processes_windows.go

```diff
@@ -10,31 +10,28 @@ import (
 	"bosun.org/opentsdb"
 )
 
-var regexes = []*regexp.Regexp{}
+var regexesProcesses = []*regexp.Regexp{}
 
 func AddProcessConfig(line string) error {
 	reg, err := regexp.Compile(line)
```

Comment:
I disagree. Panics should be reserved for truly unexpected things. Someone giving bad input is not unexpected. We should print an error and exit, as we are here. 

MustCompile is best reserved for hardcoded regexes in code that are compiled at init time (before we even look at the config file)